python/lsst/cp/pipe/cpCertify.py

```diff
@@ -155,10 +150,7 @@ def registerDatasetType(self, datasetTypeName, dataId):
             Data ID providing the list of dimensions for the new
             datasetType.
         """
-        storageClassMap = {'bias': 'ImageF',
-                           'dark': 'ImageF',
-                           'flat': 'ExposureF',
-                           }
+        storageClassMap = {'crosstalk': 'CrosstalkCalib'}
```

Comment:
Why substitute bias, dark, and flat for just crosstalk? I guess the storage classes for the former are defined somewhere else?

Reply:
All other gen3 producible calibration types should be `ExposureF`, so that's now the default.  I anticipate that as other gen2 is converted, the `storageClassMap` will need to be extended.